src/components/OSCALProfileCatalogInheritance.js

```diff
@@ -1,86 +1,53 @@
-import React, { useState } from "react";
-import { styled } from "@mui/material/styles";
-import { TreeItem, TreeView } from "@mui/lab";
+import React from "react";
+import ChevronRightIcon from "@mui/icons-material/ChevronRight";
 import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
-import ExpandLessIcon from "@mui/icons-material/ExpandLess";
-import { Grid, IconButton, List, Paper, ListSubheader } from "@mui/material";
-
-const CatalogInheritancePaper = styled(Paper)(
-  ({ theme }) => `
-  margin-top: ${theme.spacing(2)};
-  margin-bottom: ${theme.spacing(2)};
-  position: relative;
-  overflow: auto;
-`
-);
-
-function generateLabel(title, type) {
-  return type === "profile" ? `Profile: ${title}` : `Catalog: ${title}`;
-}
-
-function createTree(tree, ids) {
-  if (tree.inherited == null) {
-    return null;
-  }
-
-  const children = [];
+import { TreeItem, TreeView } from "@mui/lab";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import { OSCALSection, OSCALSectionHeader } from "../styles/CommonPageStyles";
 
-  tree.inherited.forEach((childProfileOrCatalog) => {
-    ids.push(`${childProfileOrCatalog.uuid} #${ids.length + 1}`);
-    children.push(
+export default function OSCALProfileCatalogInheritance(props) {
+  let count = 0;
+
+  const inheritedListItem = (item) => {
+    count += 1;
+
+    if (item.type === "catalog") {
+      return (
+        <TreeItem
+          key={item.uuid}
+          nodeId={count.toString()}
+          label={`Catalog: ${item.title}`}
+        />
+      );
+    }
+    return (
       <TreeItem
-        nodeId={`${childProfileOrCatalog.uuid} #${ids.length}`}
-        key={`${childProfileOrCatalog.uuid} #${ids.length}`}
-        label={generateLabel(
-          childProfileOrCatalog.title,
-          childProfileOrCatalog.type
-        )}
+        key={item.uuid}
+        nodeId={count.toString()}
```

Comment:
Out of curiosity, can we just use the `item.uuid` here? Does `nodeId` need to be an integer-ish thing? Would be nice to pull `count` out of here.

Reply:
Good point. That makes a lot more sense.